flake.nix

```diff
@@ -112,4 +118,16 @@
       };
     };
   };
+
+  nixConfig = {
+    extra-substituters = [
+      "https://cache.iog.io"
+      "https://hydra.iohk.io"
```

Comment:
Why do we need this?

Reply:
This sets the IOG caches at a flake level, so that people don't need to set it for the entire system.